packages/replay-next/src/suspense/ResumeTargetCache.ts

```diff
@@ -73,7 +73,10 @@ export const resumeTargetCache = createCache<
 
       if (frameIndex > 0) {
         const pointStack = await pointStackCache.readAsync(0, frameIndex, replayClient, pausePoint);
-        currentPoint = pointStack[frameIndex].point.point;
+        const frame = pointStack[frameIndex];
+        if (frame.point) {
+          currentPoint = frame.point.point;
+        }
```

Comment:
We should `return;` if `frame.point` is not set - otherwise the debugger will step in the top frame rather than the selected frame.

Reply:
Okay, yeah, that's what I was wondering.